lib/auth/webauthncli/fido2.go

```diff
@@ -131,28 +131,29 @@ func fido2Login(
 	var usedAppID bool
 
 	pathToRPID := &sync.Map{} // map[string]string
-	filter := func(dev FIDODevice, info *deviceInfo) (bool, error) {
+	// TODO (codingllama): not sure if capability is best name, maybe you got better suggestion.
+	deviceCapabilityCheck := func(dev FIDODevice, info *deviceInfo) error {
 		switch {
 		case uv && !info.uvCapable():
 			log.Debugf("FIDO2: Device %v: filtered due to lack of UV", info.path)
-			return false, nil
+			// TODO (codingllama): shopuld we suggest setting pin here if it has uv option present but set to false?
+			return trace.BadParameter("Device not capable due to lack of UV")
```

Comment:
Let's try and be more helpful here - is the device not capable because it's missing a PIN, and setting that would make it work, or is the device just not supported? We can take this opportunity to give better instructions.